packages/dds/tree/.eslintrc.cjs

```diff
@@ -6,7 +6,7 @@
 module.exports = {
 	extends: [require.resolve("@fluidframework/eslint-config-fluid/minimal"), "prettier"],
 	parserOptions: {
-		project: ["./tsconfig.json", "./src/test/tsconfig.json"],
```

Comment:
The tsconfig files specify which .ts files they manage, so eslint was able to figure out which parser to use where, but scoping the parserOptions for the test code alongside its rules overrides will likely make eslint's job a bit easier, and makes the config easier to reason about.